Jenkinsfile

```diff
@@ -108,231 +114,388 @@ pipeline {
-                    sh "mv zowe-node-sdk-typedoc.zip zowe-nodejs-sdk-typedoc-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    archiveArtifacts artifacts: "zowe-nodejs-sdk*-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    // Remove all tgzs after bundle is archived
-                    sh "rm -f *.tgz"
-                }
-            }
-        }
-        /************************************************************************
-         * STAGE
-         * -----
-         * Build Zowe Python SDK Bundle
-         *
-         * TIMEOUT
-         * -------
-         * 10 Minutes
-         *
-         * EXECUTION CONDITIONS
-         * --------------------
-         * - Always
-         *
-         * DECRIPTION
-         * ----------
-         * Gets the latest version of the Zowe Python SDK package from pypi.org
-         *
-         * OUTPUTS
-         * -------
-         * A Zowe Python SDK Archive.
-         ************************************************************************/
-        stage('Create Zowe Python SDK Bundle') {
-            when {
-                allOf {
-                    expression {
-                        return BRANCH_NAME.equals(MASTER_BRANCH)
+                stage('Create Bundles (Next)') {
+                    stages {
+                        /************************************************************************
+                        * STAGE
+                        * -----
+                        * Build Zowe CLI Bundle
+                        *
+                        * TIMEOUT
+                        * -------
+                        * 10 Minutes
+                        *
+                        * EXECUTION CONDITIONS
+                        * --------------------
+                        * - Always
+                        *
+                        * DESCRIPTION
+                        * ----------
+                        * Gets the latest version of the Zowe CLI and SCS plugin from Zowe
+                        * Artifactory. Creates an archive with 'fat' versions of the CLI and Plugin -
+                        *  dependencies are bundled.
+                        *
+                        * OUTPUTS
+                        * -------
+                        * A Zowe CLI Archive containing Zowe CLI and Zowe CLI Secure Credential Store Plugin
+                        ************************************************************************/
+                        stage('CLI Core (Next)') {
+                            steps {
+                                timeout(time: 10, unit: 'MINUTES') {
+                                    dir("next") {
+                                        sh "mkdir -p licenses && cd licenses && cp ../../zowe_licenses_full.zip zowe_licenses_full.zip"
+
+                                        script { zoweCliVersion = "next" }

```

Comment:
Curious if this is going to cause problems since the parallel stages are depending on the same global variable defined outside the pipeline.
Maybe we can define them in the scope of the sequential stages.

Reply:
Thanks for catching this, I think it would have caused problems 🙂 I'll stop defining these variables in the Next stages, they were left over from copying the LTS stages but aren't needed here.